src/components/GeneralInput.tsx

```diff
@@ -0,0 +1,25 @@
+import { CloudMoonRain } from 'lucide-react';
+import React from 'react';
+import { Input } from './lib/ui/input';
+import { Trykker } from 'next/font/google';
+
+type inputProps = {
+    primaryTextVisible: boolean,
+    secondaryTextVisible: boolean,
+    primaryText: string,
+    secondaryText: string,
+}
+const GeneralInput = (props: inputProps) => {
+    let primaryVisible = props.primaryTextVisible ? props.primaryTextVisible : true;
```

Comment:
Theres actually an undesired outcome here. These two lines always evaluate to true

Reply:
Roger that!